include/oneapi/dpl/internal/serial_algorithm_impl.h

```diff
@@ -18,8 +18,8 @@
 #define _ONEDPL_SERIAL_NUMERIC_IMPL_H
 
 #if (__cplusplus >= 201703L)
-#    include <algorithm>
-#    include <iterator>
+#    include <algorithm>            // KSATODO: we may include this file without __cplusplus check
+#    include <iterator>             // KSATODO: we may include this file without __cplusplus check
```

Comment:
The idea of this code block (lines 20-46) was to add a serial version of `for_each_n` when it is needed (C++17 and above) but not available (`_ONEDPL_HAS_NUMERIC_SERIAL_IMPL`). With C++17 being the minimal supported version, the algorithms is always needed, but still not always available.

Reply:
OK.